src/core/c-error.c

```diff
@@ -838,19 +840,40 @@ REBSER *Make_Error_Core(REBCNT code, REBFLG up_stack, va_list *args)
         // frame keylist is already managed)
 
         frame = Copy_Array_Shallow(root_frame);
+        SERIES_SET_FLAG(frame, SER_FRAME);
+
+        // !!! Should tweak root frame during boot so it actually is an ERROR!
+        // (or use literal error construction syntax, if it worked?)
```

Comment:
Thanks for pointing it out (and using the source line commenting).

Construction syntax is still in flux enough I'll probably do the simple thing...to have an Init_Errors() and just do a few things in there, including tweaking the VAL_TYPE for that item.  Lots of things to sweep up in this, so don't read it too religiously...it was just phase one.